src/scss/about-me.scss

```diff
@@ -1,27 +1,31 @@
-.section-about {
+.about-me {
+  &__wrapper {
     display: flex;
-    flex-direction: column;
     align-items: center;
+    flex-direction: column;
+
     padding-top: 120px;
     padding-bottom: 107px;
-}
+  }
 
-.about-title {
+  &_title {
     margin-bottom: 18px;
+
     text-align: center;
-}
+  }
 
-.text-about-me {
+  &__wrapper-text {
     display: flex;
-    flex-wrap: wrap;
-    flex-direction: column;
     align-items: center;
-}
+    flex-direction: column;
+    flex-wrap: wrap;
+  }
 
-.text-about-me span {
+  &__text {
     text-align: center;
-}
+  }
 
-.text-about-me:not(:last-child) {
+  &__wrapper-text:not(:last-child) {
```

Comment:
&__wrapper-text {
        &:not(:last-child) {
            margin-bottom: 25px;
        }
    }